easybuild/easyconfigs/v/ViennaRNA/ViennaRNA-2.0.7-goolf-1.4.10.eb

```diff
@@ -15,15 +15,15 @@ easyblock = 'ConfigureMake'
 name = 'ViennaRNA'
 version = '2.0.7'
 
-homepage = 'http://www.tbi.univie.ac.at/~ronny/RNA/vrna2.html'
+homepage = 'http://www.tbi.univie.ac.at/RNA/'
 description = """The Vienna RNA Package consists of a C code library and several
 stand-alone programs for the prediction and comparison of RNA secondary structures."""
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 toolchainopts = {'optarch': True, 'pic': True}
 
 sources = [SOURCE_TAR_GZ]
-source_urls = ['http://www.tbi.univie.ac.at/~ronny/RNA']
+source_urls = ['http://www.tbi.univie.ac.at/RNA/download/sourcecode/2_0_x/']
```

Comment:
@SethosII please avoid hardcoding (partial) versions, use the `version*` templates instead (see http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html)

in this case:

```python
source_urls = ['http://www.tbi.univie.ac.at/RNA/download/sourcecode/%(version_major)s_%(version_minor)s_x/']
```